lib/common/components/EC2InstanceCard.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import moment from 'moment'
+import Icon from '@conveyal/woonerf/components/icon'
+import React, { Component } from 'react'
+import { Panel } from 'react-bootstrap'
+import { Link } from 'react-router'
+
+import { formatTimestamp } from '../util/date-time'
+
+import type { EC2InstanceSummary } from '../../types'
+
+type Props = {
+  instance: EC2InstanceSummary,
+  showDeploymentLink?: boolean
+}
+
+export default class EC2InstanceCard extends Component<Props> {
+  _getTimeActive = () => {
+    const { instance } = this.props
+    // If instance hasn't launched, don't calculate time.
+    if (!instance.launchTime) return 'N/A'
+    // Set end time as termination time or now.
+    const endTimestamp = this._getTerminationTime()
+    let endTime = endTimestamp ? moment(endTimestamp) : moment()
+    const launchTime = moment(instance.launchTime)
+    const duration = moment.duration(endTime.diff(launchTime))
+    return duration.asDays() < 1
+      ? `${duration.asHours().toFixed(2)} hours`
+      : `${duration.asDays().toFixed(2)} days`
```

Comment:
There's already the `humanize-duration` package that is dependency for datatools that could do this.

Reply:
Ah, good call.